src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Properties.hpp

```diff
@@ -45,79 +45,94 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-#ifndef PROPERTIES_HPP
-#define PROPERTIES_HPP
+#ifndef AIRFLOWNETWORK_PROPERTIES_HPP
+#define AIRFLOWNETWORK_PROPERTIES_HPP
+
+// EnergyPlus Headers
+#include <EnergyPlus/Data/BaseData.hh>
+#include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/Psychrometrics.hh>
 
-#ifndef AIRDENSITY
-#include "../../../Psychrometrics.hh"
-#define AIRDENSITY(state, P, T, W) Psychrometrics::PsyRhoAirFnPbTdbW(state, P, T, W)
 #define AIRDENSITY_CONSTEXPR(P, T, W) Psychrometrics::PsyRhoAirFnPbTdbW(P, T, W)
 #define AIRCP(W) Psychrometrics::PsyCpAirFnW(W)
-#else
-// Need a fallback
-#endif
-
-#ifndef AIRDYNAMICVISCOSITY
-#define AIRDYNAMICVISCOSITY(T) airDynamicVisc(T)
-#else
-// Need a fallback
-#endif
-
+#define AIRDYNAMICVISCOSITY_CONSTEXPR(T) air_dynamic_viscosity(T)
 #ifndef TOKELVIN
 #include "../../../DataGlobals.hh"
 #define TOKELVIN(T) (T + DataGlobalConstants::KelvinConv)
-#else
-// Need a fallback
 #endif
 
-// EnergyPlus Headers
-#include <EnergyPlus/Data/BaseData.hh>
-#include <EnergyPlus/EnergyPlus.hh>
-
 namespace EnergyPlus {
 
 // Forward declarations
 struct EnergyPlusData;
 
 namespace AirflowNetwork {
 
-    Real64 airThermConductivity(EnergyPlusData &state, Real64 T // Temperature in Celsius
-    );
-
-    Real64 airDynamicVisc(Real64 T // Temperature in Celsius
-    );
-
-    Real64 airKinematicVisc(EnergyPlusData &state,
-                            Real64 T, // Temperature in Celsius
-                            Real64 W, // Humidity ratio
-                            Real64 P  // Barometric pressure
-    );
+    constexpr Real64 air_dynamic_viscosity(Real64 T // Temperature in Celsius
+    )
+    {
+        return 1.71432e-5 + 4.828e-8 * T;
+    }
 
-    Real64 airThermalDiffusivity(EnergyPlusData &state,
-                                 Real64 T, // Temperature in Celsius
-                                 Real64 W, // Humidity ratio
-                                 Real64 P  // Barometric pressure
-    );
+    struct AirProperties
+    {
+        AirProperties(EnergyPlusData &state) : lowerLimitErrIdx(0), upperLimitErrIdx(0), m_state(state)
+        {
+        }
+
+        Real64 density(Real64 P, // Barometric pressure
+                       Real64 T, // Temperature in Celsius
+                       Real64 W  // Humidity ratio
+        );
```

Comment:
I don't think I've ever seen this style of closing paren on its own line a la closing brace.